### DISCUSSION TITLE
how to submit new project for mentorship?

### DISCUSSION THREAD

**Discussion by parasharsingh8086** [cncf/mentoring#768] [Q&A] [answered]
Hello, I'm new to CNCF, so need some help to get started with it.

Could someone please guide me how to and where do we submit ideas for new Project proposal for mentorship in OLF or CNCF?

Your help will be greatly appreciated.

Best Regards,
Parashar

**Comment by Chinwendu20** [comment, marked as answer]
Hello @parasharsingh8086 :
I believe the info in this link would help guide you
https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

**Reply by nate-double-u** [reply]
Hi @parasharsingh8086, I've converted this issue to a discussion as it's a better forum for this kind of question.

Depending on the program you're targeting, there are different processes for submitting a project proposal. For the LFX program, @Chinwendu20 is correct, but the projects are written up and proposed by the maintainers and contributors, then applied for by potential mentees. (So, the LFX program is very project-driven).

The GSoC program has more mentee involvement; the project still submits areas of work, and potential mentees will write proposals about how they would approach the work.

Each program has instructions (or links to instructions) in this repo; I'd encourage you to look around if you're looking to apply.
